prisma/dev-seeds/user.ts

```diff
@@ -201,6 +201,17 @@ const users: User[] = [
         createdAt: new Date(),
         updatedAt: null,
     },
+    {
+        id: 10018,
+        firstName: "Greg",
+        lastName: "Maxin",
+        email: "gmaxin@uwaterloo.ca",
+        emailVerified: new Date(),
+        role: roles.PROGRAM_ADMIN,
+        image: null,
+        createdAt: new Date(),
+        updatedAt: null,
+    },
```

Comment:
I'm going to remove this for now. In general, if you want to bootstrap you accounts, please just change the existing users with your email. eg:  `ricksonyang+admin@uwblueprint.org` -> `gmaxin+admin@uwaterloo.ca` same vice versa with volunteer users